### ISSUE TITLE
Leveraging National Metadata fields from US site

### ISSUE THREAD

**Issue by dawncomer** [dawncomer/open-sdg-site-starter#27]
Hi @Brockfanning,  I saw some national metadata fields on the US SDG site that I'm not seeing on our local site (see attached).  Is there a way to bring in some of these fields and/or make the fields on our platform?
Specifically, the fields highlighting, 'comments and limitations' 'disaggregation categories', 'method of computation'.
Thanks!

![image](https://user-images.githubusercontent.com/46500838/56003119-0f104200-5c7a-11e9-9912-e7def81fc9e1.png)

**Comment by dawncomer** [comment]
@brockfanning
Link to above image.... https://sdg.data.gov/5-b-1/

**Comment by brockfanning** [comment]
Hi @dawncomer yes absolutely. The first step would be to add the fields to [this file](https://github.com/dawncomer/open-sdg-data-starter/blob/develop/_prose.yml). The syntax is a little quirky and the indentation is important, but you can copy-paste from an entry to create something new.

Once it has been added to this "schema", you can start using the new field in actual metadata.

**Comment by dawncomer** [comment]
ok thanks @brockfanning i'll work on this as well

**Comment by dawncomer** [comment]
Hi @brockfanning , I've been able to make updates to the fields.   Next question, for [11.7.1, I've entered content in the Definitions field](http://prose.io/#dawncomer/open-sdg-data-starter/edit/develop/meta/11-7-1.md) but for some reason it is not showing on the site.  Any insights as to what I may be missing?  Thanks!

![image](https://user-images.githubusercontent.com/46500838/57496801-ae305580-7289-11e9-80f5-ad837ad912bd.png)

**Comment by brockfanning** [comment]
@dawncomer I think it's just a timing thing. One of the most non-obvious quirks of this platform is the interaction between the data repo and the site repo. The site repo, when it gets "built", consults the data repo. So, before the site repo can show any change, the data repo needs to be changed first.

But the catch is, the site repo does not automatically get built when the data repo gets built. In other words, making a change to the data repo, alone, will not produce any change to the site repo. This the reason for the nightly build of the site repo.

Long story short, you should see the change after the next nightly build of the site repo. Or, you could also "trigger" a build by making any commit to the site repo.

If you're ever not sure of the timing, you could look at the CircleCI dashboard and compare the times of the builds. Note how the data repo builds all happened after the last site repo build: https://circleci.com/gh/dawncomer

**Comment by brockfanning** [comment]
@dawncomer One more note about your recent changes to _prose.yml: To be on the safe side, you might want to make the "name" of each new field limited to lowercase letters, numbers, and underscores. So for example, instead of "My Field Name", something like "my_field_name". I don't know for sure whether this is 100% necessary, but that has been our approach with the existing fields.

The catch is that this will leave some "orphaned" fields in your metadata. To avoid this problem I just created a PR to adjust _prose.yml and all the metadata at the same time: https://github.com/dawncomer/open-sdg-data-starter/pull/7

**Comment by dawncomer** [comment]
Hi @brockfanning thanks - FYI, I did trigger a a nightly build and the problem persisted.  I'll make sure names are all lower case, etc and try the PR and see if that makes a difference.

**Comment by dawncomer** [comment]
@brockfanning it seems making updates to the 'name' of each to field to lower case made a difference.  thanks

**Comment by brockfanning** [comment]
Cool, good to hear.
